ansible/library/extract_log.py

```diff
@@ -100,6 +98,7 @@ def extract_lines(directory, filename, target_string):
 
     return result
 
+
```

Comment:
pep8: sinle line between methods

Reply:
But its a function, not method. I have added this blank line because all other functions in this file have 2 blank lines, and only this was missed.
And according to pep8 - "Surround top-level function and class definitions with two blank lines."

Reply:
I see, thanks